client/vite.config.ts

```diff
@@ -49,6 +50,17 @@ if (platform === 'web') {
   plugins.push(wasmPack([{ path: '../bindings/web/', name: 'libparsec_bindings_web' }]));
 }
 
+if (process.env.VITE_SENTRY_AUTH_TOKEN) {
+  const sentryPlugin = sentryVitePlugin({
+    org: 'scille',
+    project: 'parsec3-frontend',
+    authToken: process.env.VITE_SENTRY_AUTH_TOKEN,
```

Comment:
```suggestion
if (process.env.SENTRY_AUTH_TOKEN) {
  const sentryPlugin = sentryVitePlugin({
    org: 'scille',
    project: 'parsec3-frontend',
    authToken: process.env.SENTRY_AUTH_TOKEN,
```

Reply:
Vite blocks any env variable that is not prefixed by `VITE_` (to avoid exposing variables to the app)